envydis/gm107.c

```diff
@@ -643,6 +646,17 @@ static struct insn tabeed0sz[] = {
 	{ 0, 0, OOPS },
 };
 
+static struct insn tabeec8_0[] = {
```

Comment:
Don't do this — this breaks the assembler, and envydis natively supports this encoding in proper registers (use `.xorend=7` when defining the bitfield).